### PR TITLE
feat: add tracking and update styles for loan tags

### PR DESCRIPTION
**Pull request by egoroza**

### CODE REVIEW

File: src/components/LoanCards/GridLoanCard.vue [hunk 1]
```diff
@@ -164,6 +164,13 @@ export default {
 		},
 		trackInteraction(args) {
 			this.$emit('track-interaction', args);
+			if (args?.interactionType === 'addToBasket' && this.getTagInfo()) {
+				this.$kvTrackEvent(
+					'loan-card',
+					'add-to-basket',
+					this.getTagInfo()
```
Comment (eddieferrer) [inline src/components/LoanCards/GridLoanCard.vue:171]: We analytics labels should always be kebab-case -- so these should be 'ending-soon' etc

File: src/components/LoanCards/LoanTags/LoanTag.vue [hunk 2]
```diff
@@ -39,6 +39,13 @@ export default {
 			default: 'endingSoon'
 		}
 	},
+	mounted() {
+		this.$kvTrackEvent(
+			'loan-card',
+			'view-tag',
```
Comment (eddieferrer) [inline src/components/LoanCards/LoanTags/LoanTag.vue:45]: 'view-tag' is not in the web analytics action list. Can you provide a screenshot or link to this feature so I can suggest analytics for this? Not sure where these tags are

File: src/components/LoanCards/LoanTags/LoanTag.vue [hunk 3]
```diff
@@ -39,6 +39,13 @@ export default {
...
+			this.variation
```
Comment (eddieferrer) [inline src/components/LoanCards/LoanTags/LoanTag.vue:46]: This has to be kebab-case

File: src/components/LoanCards/LoanTags/LoanTag.vue (hunk 2 above)
Comment (egoroza) [reply src/components/LoanCards/LoanTags/LoanTag.vue:45]: They look like these https://github.com/kiva/ui/pull/4336

Comment (egoroza) [reply src/components/LoanCards/LoanTags/LoanTag.vue:45]: can change to `'loan-card', 'show', 'view-loan-tag', this.variation`

Comment (eddieferrer) [reply src/components/LoanCards/LoanTags/LoanTag.vue:45]: Let's go with: ['loan-card', 'show', 'tag-almost-funded']

**Review by mcstover — APPROVED** [review]
Thank you both for working to align with the new analytics spec

File: src/components/LoanCards/LoanTags/LoanTag.vue (hunk 2 above)
Comment (egoroza) [reply src/components/LoanCards/LoanTags/LoanTag.vue]: sgtm ty!!

**Review by roger-in-kiva — APPROVED** [review]

**Pull request merged**